openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -602,8 +607,11 @@ In addition to the response parameters defined in [@!RFC6749], the AS MAY return
 
 * `c_nonce`: OPTIONAL. JSON string containing a nonce to be used to create a proof of possession of key material when requesting a Credential (see (#credential_request)). When received, the Wallet MUST use this nonce value for its subsequent Credential Requests until the Credential Issuer provides a fresh nonce.
 * `c_nonce_expires_in`: OPTIONAL. JSON integer denoting the lifetime in seconds of the `c_nonce`.
+* `identifiers`: OPTIONAL. JSON array of JSON strings that each identify a credential that can be issued using Access Token returned in the same response. MUST NOT be used when `authorization_details` parameter was used to request issuance of a certain Credential type as defined in (#authorization-details). 
```

Comment:
A wallet could use `scope` and `authorization_details` in the same request. So the language should be along the lines of "this array contains the credentials that can be requested based on the scope values passed to the authorization request". This means `identifiers` and `authorization_details/identifiers` can turn up in the same token response.

Reply:
```suggestion
* `identifiers`: OPTIONAL. JSON array of JSON strings that each identify a credential that can be issued using Access Token returned in this response. 
```
I am sorry, i cannot understand a proposed sentence. I think removing `MUST NOT...` sentence should be sufficient.